web/src/user_groups.ts

```diff
@@ -90,8 +100,9 @@ export function update(event: UserGroupUpdateEvent): void {
         group.name = event.data.name;
         user_group_name_dict.set(group.name, group);
     }
-    if (event.data.description !== undefined) {
+    if (event.data.description !== undefined && event.data.rendered_description !== undefined) {
         group.description = event.data.description;
+        group.rendered_description = event.data.rendered_description;
```

Comment:
I guess `clean_up_group_description` needs to be called here as well for the new value.

Reply:
There is no need to update the `event.data.rendered_description` field here, doing `group.rendered_description = clean_up_group_description(event.data.rendered_description)` will be better.